MacroChallenge/MacroChallenge/inventoryViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  inventoryViewController.swift
+//  MacroChallenge
+//
+//  Created by Lucas Ronnau on 02/07/20.
+//  Copyright © 2020 Pedro Vargas. All rights reserved.
+//
+
+import UIKit
+
+class inventoryViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    
+
+    @IBAction func openInventory(_ sender: Any) {
+        
+        if let inventoryV = Bundle.main.loadNibNamed("inventory", owner: self, options: nil)?.first as? inventoryView {
+            
+            inventoryV.closeView.addTarget(self, action: #selector(inventoryViewController.closeInventory(sender:)), for: .touchUpInside)
+            
+            self.view.addSubview(inventoryV)
+        }
+        
+    }
+    
+    @objc func closeInventory (sender: UIButton) {
+        
+        let viewPosition = self.view.subviews.endIndex - 1
+        self.view.subviews[viewPosition].removeFromSuperview()
+        
```

Comment:
Acho que isso funcionaria
```suggestion
        self.dismiss(animated: false, completion: nil)
        
```